File: 01-Assignment-02-02-2026/Using_C/program_3.c
/* Program 3: Find the area of a triangle given three sides */

#include <stdio.h>
#include <math.h>

int main() {
    float a, b, c, area, s;

    printf("Enter three sides of the triangle:\n");
    scanf("%f %f %f", &a, &b, &c);

    // Check for valid triangle
    if (a + b > c && a + c > b && b + c > a) {

        // Heron's formula
        s = (a + b + c) / 2;
        area = sqrt(s * (s - a) * (s - b) * (s - c));

        printf("Area of the triangle = %.2f\n", area);
    } 
    else {
        printf("Invalid triangle\n");
    }

    return 0;
}

File: 01-Assignment-02-02-2026/Using_Java/Program3.java
/*
Program 3
Java program to find the area of a triangle whose three sides are given
*/

import java.util.Scanner;

public class Program3 {
    /* Solution ->
    When lenght of each side of triange is given the Area(A)
    A = sqrt(s*(s-a)*(s-b)*(s-c))
    where , s is semi-perimeter 
    s = (a+b+c) / 2 */


    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double a, b, c, s, Area;

        System.out.print("Enter the Length a : ");
        a = scanner.nextDouble(); 
        
        System.out.print("Enter the Length b : ");
        b = scanner.nextDouble(); 
        
        System.out.print("Enter the Length c : ");
        c = scanner.nextDouble(); 

        s = (a + b + c) / 2.0; 
        
        Area = Math.sqrt(s * (s - a) * (s - b) * (s - c));
        
        System.out.println("Area of Triangle : " + Area);
        scanner.close();
    }
}





